packages/deployments/contracts/contracts/core/connext/libraries/SwapUtils.sol

```diff
@@ -208,19 +206,18 @@ library SwapUtils {
       // if i == tokenIndex, dxExpected = xp[i] * d1 / d0 - newY
       // else dxExpected = xp[i] - (xp[i] * d1 / d0)
       // xpReduced[i] -= dxExpected * fee / FEE_DENOMINATOR
-      xpReduced[i] = xpi.sub(
-        ((i == tokenIndex) ? xpi.mul(v.d1).div(v.d0).sub(v.newY) : xpi.sub(xpi.mul(v.d1).div(v.d0)))
-          .mul(v.feePerToken)
-          .div(FEE_DENOMINATOR)
-      );
+      xpReduced[i] =
+        xpi -
+        ((((i == tokenIndex) ? ((xpi * v.d1) / v.d0 - v.newY) : (xpi - (xpi * v.d1) / v.d0)) * v.feePerToken) /
+          FEE_DENOMINATOR);
```

Comment:
what is your suggestion? @0xbok

Reply:
it might be a personal preference, but I think it'll be easier to parse by eyes if `* v.feePerToken` is in a different line. But feel free to ignore this comment entirely.